editors/frescobaldi/Portfile

```diff
@@ -24,9 +24,16 @@ checksums           rmd160  3717c40f177762877757037d1887c409315e47be \
                     size    6567483
 
 subport ${name}-devel {
+    set devel_version   ${github.version}-dev
+    github.setup        wbsoft frescobaldi c2b5af3e1cdcde51f996407a27cebff4ad9e1d33
     conflicts           ${name} ${name}2
-    version             20200413
-    set devel_version   ${github.version}
+    version             20200719
+
+    github.tarball_from tarball
```

Comment:
Is this essential? We would rather you use `github.tarball_from archive` if possible. This will require changing the checksums.

Reply:
I simply want to undo the setting of `github.tarball_from` in this subport. I used `tarball` because it is the default in the `github` PortGroup.

https://github.com/macports/macports-ports/blob/2c6b50f278633cf2b2172a7fa618a5ae67cb0d9c/_resources/port1.0/group/github-1.0.tcl#L24

Why is `archive` preferred? It is strange that the default value is not the preferred value. Moreover, even the error message for the `tags` value suggests to use `tarball`.

https://github.com/macports/macports-ports/blob/2c6b50f278633cf2b2172a7fa618a5ae67cb0d9c/_resources/port1.0/group/github-1.0.tcl#L45

Reply:
Anyway, I should probably remove that line and guard the `github.tarball_from` setting in the main port with an appropriate `if`, instead.
@ryandesign What do you think?

Reply:
> Why is `archive` preferred?

Because that is the new style of automatically generated tarball. It is the style linked to from the GitHub web interface these days and for the past several years. It has an advantage over the old "tarball" style in that the checksums will not change if the repository is moved to a new owner; this problem has happened to us countless times so we would like to avoid it as much as possible by not using the old style tarballs anymore.

> It is strange that the default value is not the preferred value.

The "archive" type did not exist when the github portgroup was first created. We cannot change the default now without breaking all the ports that already use the portgroup and do not specify the tarball type.

> Moreover, even the error message for the `tags` value suggests to use `tarball`.

That is correct. The portgroup used to use the word "tags" to describe the old-style automatically generated tarball, and now uses the word "tarball" to describe it.

> Anyway, I should probably remove that line and guard the `github.tarball_from` setting in the main port with an appropriate `if`, instead.

You should use the new-style automatically generated tarball if possible by specifying `github.tarball_from archive`.

Reply:
Done.